src/components/InvertedFlatList/BaseInvertedFlatList/index.tsx

```diff
@@ -1,7 +1,7 @@
 import type {ForwardedRef} from 'react';
 import React, {forwardRef, useMemo} from 'react';
+import {FlatList} from 'react-native';
 import type {FlatListProps, ScrollViewProps} from 'react-native';
-import FlatList from '@components/FlatList';
```

Comment:
Because previously the `index.ts` file looked like that

```
import {FlatList} from 'react-native';

export default FlatList;
```

so we were using the basic `FlatList` component here.
Now this file contains a modified version of FlatList, so we need to use the base one here for this component to stay unchanged

Reply:
But the component used would be different given the platform being used, right? Before we was using `FlatList` from '@components/FlatList', that its `index.ts` was just point to the core `FlatList`, but if the user was using android it would use `index.android.tsx`, if web it would use `index.web.js`.

Now it will only use the core `FlatList`, it seems wrong to me.

Reply:
You're right! Fixed